Dockerfile

```diff
@@ -1,18 +1,33 @@
 FROM golang:1.21.4-alpine3.18 AS build-env
 
+ARG DB_BACKEND=goleveldb
+ARG ROCKSDB_VERSION="8.5.3"
```

Comment:
Looks like this `ARG` is not used in Dockerfile, do we need to pass it to `make` command?

Reply:
yeah, the make command will pick it up. Allows to specify the rocksdb version to build when building the docker image